twelvelabs/resources/search.py

```diff
@@ -81,9 +82,9 @@ def query(
                 files["query_media_file"] = file
             else:
                 files["query_media_file"] = query_media_file
-        else:
-            # Request should be sent as multipart-form even file not exists
-            files["dummy"] = ("", "")
+
+        if query_media_url is not None:
+            files['query_media_url'] = BytesIO(query_media_url.encode())
```

Comment:
have you checked working with providing `query_text`, not `query_media_file` or `query_media_url`?
the reason why I added meaningless "dummy" into files was that http library detects whether it's multiform request by `files` existence.

Reply:
Actually, it looks like `Search.query()` and `EmbedTask.create` work ok with the url arguments - it's just `Task.create` that is broken. I'll remove the changes in those two files and re-push.

Reply:
Thanks for addressing. LGTM!